src/AddressOwnershipTool.Tests/BlockExplorerTests.cs

```diff
@@ -0,0 +1,19 @@
+using Xunit;
+
+namespace AddressOwnershipTool.Tests
+{
+    public class BlockExplorerClientTests
+    {
+        [Fact]
+        public void GivenAddressWIthFunds_WhenCheckingIfHasBalance_ThenTrueIsReturned()
+        {
+            var address = "Sdm4cmKhBtKehvXkpRSVaTX1RAjsE5CVNc";
+
+            var blockExplorerClient = new BlockExplorerClient();
+
+            var result = blockExplorerClient.HasBalance(address);
```

Comment:
I am not sure that tying this test to the existence of an externally-hosted API is a good idea

Reply:
Removed